docs/cql/ddl.rst

```diff
@@ -544,6 +544,11 @@ options of a table are described in the following sections.
 Compact tables
 ``````````````
 
+Support for compact tables has been deprecated.
+By default, newly created tables cannot be defined with the ``COMPACT STORAGE`` option.
+To enable this feature, set the ``enable_create_table_with_compact_storage`` option to ``true`` in ``scylla.yaml``,
+yet beware that support for compact tables is planned to be permanently removed in a future version.
+
 ..
    .. caution:: Since Apache Cassandra 3.0, compact tables have the exact same layout internally than non compact ones (for the
       same schema obviously), and declaring a table compact **only** creates artificial limitations on the table definition
```

Comment:
What's the garbage below "Unchanged files with check annotations Beta"?

Reply:
I don't understand what you're referring to.
Is it something generated by github or something in ddl.rst?

Reply:
It looks like github actions is warning about spelling mistakes in comments

Reply:
It does, and you can ignore it.

Reply:
What's less clear to me is how does it still complain on errors I had hoped were fixed by me over multiple PRs :-/